csharp/test/Drivers/Databricks/E2E/DatabricksTestConfiguration.cs

```diff
@@ -45,5 +45,8 @@ public class DatabricksTestConfiguration : SparkTestConfiguration
 
         [JsonPropertyName("traceStateEnabled"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
         public string TraceStateEnabled { get; set; } = string.Empty;
+
+        [JsonPropertyName("isCITesting"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public bool isCITesting { get; set; } = false;
```

Comment:
```suggestion
        public bool IsCITesting { get; set; } = false;
```
nit: follow C# convention of UpperCase property names.